### DISCUSSION TITLE
Measures in diagrams

### DISCUSSION THREAD

**Discussion by dbronk** [TabularEditor/TabularEditor3#381] [🙋♀️TE3 Q&A]
New to Tabular Editor and the main reason we are looking to get v3 is so that we can create nice visual diagrams of our models (ie: erd's).  While this feature is pretty slow and does not give much control, so far it looks like it may suit our purpose, except for one thing.  I can't seem to figure out how to show measures in the diagram.  Anyone have advice?

Thanks.

**Comment by otykier** [comment]
While we do have plans to address the performance and general “look and feel” of TE3 diagrams, we are not planning to add measures. This is a deliberate design decision, to reduce cluttering of the diagram view, as it is not uncommon for tables to have hundreds of measures. Moreover, measures are not relevant for understanding the structure of the model or creating relationships between tables, which is the primary purpose of the diagram view.

Of course if users insist on wanting measures in the diagram view, we might reconsider, but this is the first time we’ve had this request, and it is not clear to us what the benefit of this would be.
